pt-BR/mail/chrome/messenger/preferences/compose.dtd

```diff
@@ -43,103 +43,103 @@
 <!ENTITY directories.accesskey                 "S">
 <!ENTITY editDirectories.label                 "Editar diretórios…">
 <!ENTITY editDirectories.accesskey             "E">
-<!ENTITY matchText.label                       "If there is a match in your local address books:">
-<!ENTITY skipDirectory.label                   "Do not search in the directory">
+<!ENTITY matchText.label                       "Se já houver um nos seus catálogos locais de endereços:">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY matchText.label                       "Se já houver correspondência nos seus catálogos locais de endereços:">